src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -86,39 +95,54 @@ $range-thumb-shadow-focus: 0 0 0 rem(1px) color('indigo');
   width: $range-thumb-size;
   height: $range-thumb-size;
```

Comment:
Thumbs on Safari don't appear to be perfect circles for some reason

![image](https://user-images.githubusercontent.com/22782157/73109001-f357ee80-3ecf-11ea-8afe-b33ee7b90011.png)

Reply:
That’s weird

Reply:
In the "Layout" tab in the sidebar in FF there is a box model for each element. Taking a look at it shows that the thumb has a border and padding. For this, the element should be `16x16` with not padding or border.

<img width="490" alt="Screen Shot 2020-01-27 at 3 18 11 PM" src="https://user-images.githubusercontent.com/344839/73222676-ac106e80-4118-11ea-980c-6fe0de640428.png">

Reply:
Resolved with https://github.com/Shopify/polaris-react/pull/2655/commits/a298747542ebb27e5701658306920208d0b6703d